{"uuid": "0acc7c42-00f3-4d6d-bde7-1c8f4bd987a8", "children": ["6859c9cd-42a0-40fc-a00a-3aa5fc8dc0c0", "8d9fe69f-310a-4302-816c-3995db1769b7"], "befores": [{"name": "record", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [WinError 2] 系统找不到指定的文件。\n", "trace": "  File \"c:\\testtools\\python\\python37\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\testtools\\python\\python37\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\testtools\\python\\python37\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"C:\\Users\\heli\\PycharmProjects\\diy_appium\\scripts\\conftest.py\", line 19, in record\n    p= subprocess.Popen(cmd,shell=False,stdout=subprocess.PIPE,stderr=subprocess.STDOUT)\n  File \"c:\\testtools\\python\\python37\\lib\\subprocess.py\", line 800, in __init__\n    restore_signals, start_new_session)\n  File \"c:\\testtools\\python\\python37\\lib\\subprocess.py\", line 1207, in _execute_child\n    startupinfo)\n"}, "start": 1592380906897, "stop": 1592380906905}], "start": 1592380906897, "stop": 1592380907039}